### PR TITLE
PADV-704 Add instructors component

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-704

### Description
Add instructors option in menu sidebar, instructors data table

### Changes made

- [ ] Add instructors option in menu sidebar,
- [ ] Add instructors data table
- [ ] Add tests
- [ ] Align page title with table 
 
### Screenshots
image

### How to test

- Clone the Institution Portal MFE
- Run npm install.
- Run npm run start
- Go to http://localhost:1980/instructors

### CODE REVIEW

File: src/features/Instructors/InstructorsPage/_test_/reducer.test.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import {
+  FETCH_INSTRUCTOR_DATA_REQUEST,
+  FETCH_INSTRUCTOR_DATA_SUCCESS,
+  FETCH_INSTRUCTOR_DATA_FAILURE,
+  UPDATE_CURRENT_PAGE,
+} from '../../actionTypes';
```
Comment (anfbermudezme) [inline src/features/Instructors/InstructorsPage/_test_/reducer.test.jsx:6]: Could you please use absolute imports and implement it in all the components?

File: src/features/Instructors/InstructorsPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+const InstructorsPage = () => {
+  const [state, dispatch] = useReducer(reducer, initialState);
+  const [currentPage, setCurrentPage] = useState(1);
+
+  const fetchData = async () => {
+    dispatch({ type: FETCH_INSTRUCTOR_DATA_REQUEST });
+
+    try {
+      const response = await getInstructorData(currentPage);
```
Comment (anfbermudezme) [inline src/features/Instructors/InstructorsPage/index.jsx:36]: Could you please use CamelCaseobject? https://github.com/openedx/frontend-platform/blob/master/src/utils.js#L73. An example of use is https://github.com/Pearson-Advance/frontend-app-institution-portal/blob/master/src/features/Students/StudentsPage/index.jsx#L99.

File: src/features/Instructors/InstructorsTable/columns.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const columns = [
+  {
+    Header: 'User Name',
+    accessor: 'instructor_username',
+  },
+  {
+    Header: 'Name',
+    accessor: 'instructor_name',
```
Comment (anfbermudezme) [inline src/features/Instructors/InstructorsTable/columns.jsx:8]: The accessors using CamelCaseObject will be change. Ex: `instructorName`

**Review by anfbermudezme — CHANGES_REQUESTED** [review]

**Review by anfbermudezme — APPROVED** [review]
LGTM

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**